schema/core.md

```diff
@@ -0,0 +1,193 @@
+  "url": "https://www.hydroshare.org/resource/9d413b9d57824a79b8239a5f7c4fdf51/",
+  "dateCreated": "2022-08-16T17:35:21+00:00",
+  "author": {
+    "@list":[
+      {
+        "@type": "Organization",
+        "name": "U.S. Geological Survey (USGS)"
+      },
+      {
+        "@type": "Organization",
+        "name": "NOAA National Weather Service"
+      },
+      {
+        "@type": "Organization",
+        "name": "Texas Commission for Environmental Quality (TCEQ)"
+      },
+      {
+        "@type": "Person",
+        "name": "David Arctur",
+        "email": "david.arctur@utexas.edu",
+        "identifier": {
+          "@id": "https://orcid.org/0000-0002-4708-6302",
+          "@type": "PropertyValue",
+          "propertyID": "https://registry.identifiers.org/registry/orcid",
+          "url": "https://orcid.org/0000-0002-4708-6302",
+          "value": "0000-0002-4708-6302"
+        }
+      }
+    ]
+  }
+}
+```
+
+
+
+**Recommended Metadata**
+
+|Property|Class|Type|Cardinality|Description| 
+|---|---|---|---|---|
+|hasPart|CreativeWork|CreativeWork|0+|Indicates an item or CreativeWork that is part of this item|
+|isPartOf|CreativeWork|CreativeWork OR URL |0+|Indicates an item or CreativeWork that this item, or CreativeWork (in some sense), is part of.|
+|associatedMedia|CreativeWork|MediaObject|0+| A media object that encodes this CreativeWork. This property is a synonym for encoding.|
+|creativeWorkStatus | CreativeWork | DefinedTerm \| Text | 0,1 | The status of a creative work in terms of its stage in a lifecycle. Example terms include Incomplete, Draft, Published, Obsolete. Some organizations define a set of terms for the stages of their publication lifecycle.|
+|dateModified |	CreativeWork |Date \| DateTime | 0,1| The date on which the CreativeWork was most recently modified or updated. | 
+|keywords | CreativeWork | DefinedTerm  \| Text \| URL	| 0+ |Keywords or tags used to describe the dataset, delimited by commas.|
+|funding| CreativeWork | Grant | 0+ | A Grant that directly or indirectly provide funding or sponsorship for creation of the dataset.|
+
+
+The following is an example implementation of recommended metadata
+properties for a SchemaOrg Dataset item:
+
+<pre lang="json">
```

Comment:
This is just for rendering the code snippet.

Reply:
I get it. Viewers will just see the snippet and not these tags. Github markdown has the ability to do syntax highlighting on fenced code blocks . It looks like some of the examples use this, but others use the HTML tags. Given that JSON-LD is embedded in HTML landing pages, I suggest using HTML in your examples sparingly (if at all) to avoid any confusion.